packages/data-context/src/DataContext.ts

```diff
@@ -184,7 +205,14 @@ export class DataContext {
 
   @cached
   get cloud () {
-    return new CloudDataSource(this)
+    return new CloudDataSource({
+      fetch: (...args) => this.util.fetch(...args),
+      getUser: () => this.user,
+      logout: () => this.actions.auth.logout().catch(this.logTraceError),
+      onError: (err) => {
+        return this.onError(getError('DASHBOARD_GRAPHQL_ERROR', err), 'Cypress Dashboard Error')
+      },
```

Comment:
Need to revisit whether this is actually what we want to do, or if we should just be putting `<Alert />` components in various places